### ISSUE TITLE
If I have two seperate networks, how to add one network's loss to antoher network during training?

### ISSUE THREAD

**Issue by yushuinanrong** [keras-team/keras#7998]
Hi all,

I'm trying to train two networks A and B. Ideally, B is optimized according to its own loss plus the loss of A (for every iteration).
For some reasons, I cannot merge the two networks.
I tried using an API model and thus the loss of this model is the sum of loss A and loss B (you can see the model structure using the url at bottom). 
To check if the model really work as I expect,  I set the weight of loss of A and B to be 0 and 1 respectively. In such case, training network A should only depends on the loss of network B. 
However, if I do so, network A cannot be trained at all. 
Any idea?

https://user-images.githubusercontent.com/17254532/30920739-b9d60156-a372-11e7-8b1e-3e5f8cddf709.png
